typescript-carousel/main.ts

```diff
@@ -0,0 +1,78 @@
+const imgRelativeUrls = [
+  './images/001.png',
+  './images/004.png',
+  './images/007.png',
+  './images/025.png',
+  './images/039.png',
+];
+const $indicators = document.querySelector('#indicators');
+if (!$indicators) throw new Error('$indicators not found!');
+const indicatorList = Array.from(
+  $indicators.querySelectorAll('i')
+) as HTMLElement[];
```

Comment:
Did you need to create an array from a NodeList here? Couldn't you just use the NodeList and iterate through it? 🤔